integrations/schemas/curations-schema.json

```diff
@@ -85,30 +65,10 @@
               },
               "hash": {
                 "$ref": "#/definitions/hash"
-              },
-              "hash_algorithm": {
-                "type": "string"
-              }
-            },
-            "if": {
-              "properties": {
-                "hash": {
-                  "type": "object"
-                }
               }
             },
-            "then": {
-              "required": [
-                "hash"
-              ]
-            },
-            "else": {
-              "required": [
-                "hash",
-                "hash_algorithm"
-              ]
-            },
             "required": [
+              "hash",
```

Comment:
I believe this commit should also drop the `"string"` type from https://github.com/oss-review-toolkit/ort/blob/ca7beff23342d5c1b9a2bdbaa6c79c9e6cadadeb/integrations/schemas/curations-schema.json#L133.